crates/forge/src/scarb.rs

```diff
@@ -36,23 +41,70 @@ pub fn build_contracts_with_scarb(filter: PackagesFilter) -> Result<()> {
 
 pub fn build_test_artifacts_with_scarb(filter: PackagesFilter) -> Result<()> {
     ScarbCommand::new_with_stdio()
-        .arg("snforge-test-collector")
+        .arg("build")
+        .arg("--test")
         .packages_filter(filter)
         .run()
         .context("Failed to build test artifacts with Scarb")?;
     Ok(())
 }
 
+/// collecting by name allow us to dedup targets
+/// we do it because they use same sierra and we display them without distinction anyway
+fn test_targets_by_name(package: &PackageMetadata) -> HashMap<String, &TargetMetadata> {
+    fn test_target_name(target: &TargetMetadata) -> String {
+        // this is logic copied from scarb
```

Comment:
Would make sense to link relevant file

Reply:
Probably will be outdated before a new js framework is released

Reply:
You can do permanent links on gh, it links to a specific commit

<img width="290" alt="image" src="https://github.com/user-attachments/assets/7431aa96-f5bf-4e69-9035-11e5d169d000">